R/migrations.R

```diff
@@ -43,9 +43,10 @@ migrate_task <- function(task_id, queue, dry_run = FALSE) {
   new_res <- migrate(res)
   if (!dry_run) {
     store <- rrq:::rrq_object_store(queue$queue$con,
-                                    queue$queue$keys)
+                                    r6_private(queue$queue)$keys)
```

Comment:
Do we still need this migration support? Or has everything been migrated now?

Reply:
It's all been migrated, I'll take that out